spec/index.bs

```diff
@@ -15884,7 +15882,12 @@ It guarantees that:
 
 ### Sample frequency shading ### {#sample-frequency-shading}
 
-<p class="note editorial"><span class=marker>Editorial note:</span> fill out the section
+For multisampled render attachments fragment shaders can be run once per-pixel or once per-sample. Fragment shaders will
+run once per-sample if the `sample_index` [=builtin=] is used by the shader. Otherwise fragment shaders are run once
+per-pixel with the result broadcast out to each of the samples included in the [=final sample mask=].
```

Comment:
Using sample interpolation also triggers per-sample shading.

I think #4696 still needs a bit more investigation before we can fully commit here. The most accurate I think we can say is, if sample_index _or_ sample interpolation is used _and_ contributes to the output of the shader (can't be optimized away), then shading **must** be per-sample, and otherwise shading **may** be per-pixel? That matches closer to reality right now.

Reply:
Updated. Let me know what you think of the new lanugage.